internal/converter/internal/test_common/testing.go

```diff
@@ -45,7 +45,7 @@ const (
 func TestDirectory(t *testing.T, folderPath string, sourceSuffix string, loadFlowConfig bool, extraArgs []string, convert func(in []byte, extraArgs []string) ([]byte, diag.Diagnostics)) {
 	require.NoError(t, filepath.WalkDir(folderPath, func(path string, d fs.DirEntry, _ error) error {
 		if d.IsDir() {
-			return nil
+			return filepath.SkipDir
 		}
```

Comment:
This has the unfortunate side effect of skipping _all_ tests; the first callback in filepath.WalkDir when walking testdata will be the testdata directory itself; by skipping that directory, all test cases are accidentally skipped.

Reply:
I'd be guilty not fixing this :D. Now only child directories are skipped and not the top-level one